src/scms/hg.ts

```diff
@@ -1,18 +1,13 @@
 import path from 'path'
 
-import findUp from 'find-up'
-import { Output, exec } from 'tinyexec'
+import * as find from 'empathic/find'
+import { exec, Output } from 'tinyexec'
 
 export const name = 'hg'
 
 export const detect = (directory: string) => {
-  const hgDirectory = findUp.sync('.hg', {
-    cwd: directory,
-    type: 'directory',
-  })
-  if (hgDirectory) {
-    return path.dirname(hgDirectory)
-  }
+  const found = find.up('.hg', { cwd: directory })
```

Comment:
What if there is a `.hg` file?

Reply:
It's always a directory. If you have a `.hg` file your mercurial project would be broken

Reply:
That just means there is a `.hg` file and it's not a mercurial project.

Reply:
We can stat the result to check if it's a directory, tho the odds of this happening is very low.